### PR TITLE
Refine attributed string texts

### PR DESCRIPTION
**Pull request by bluwave**
refs: MBL-14413
affects: Parent
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/p54jpbb) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fp54jpbb)
teacher: [Link](https://rebrand.ly/90n8edg) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F90n8edg)
parent: [Link](https://rebrand.ly/1igagz8) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2F1igagz8)

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Parent

[MBL-14413](https://instructure.atlassian.net/browse/MBL-14413)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.62% | 93.62% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Parent/Parent/Create Account/CreateAccountViewController.swift [hunk 1]
```diff
@@ -165,6 +165,43 @@ class CreateAccountViewController: UIViewController, ErrorViewController {
...
+            .link,
+            value: "https://www.instructure.com/policies/acceptable-use?newhome=canvas",
+            range: (message as NSString).range(of: terms)
+        )
+        attributed.addAttribute(
+            .link,
+            value: "https://www.instructure.com/policies/privacy?newhome=canvas",
+            range: (message as NSString).range(of: privacyPolicy)
+        )
+        return attributed
```
Comment (vanwagonet) [inline Parent/Parent/Create Account/CreateAccountViewController.swift:191]: It's not fun how much code this takes to do, but I think the result will be best for translators & end users. Strong work.

Comment (armstrongnate) [reply Parent/Parent/Create Account/CreateAccountViewController.swift:191]: Wow. Nice work. I'm surprised it's this complicated to get some tappable text in UIKit.

**Pull request merged**